.github/workflows/ci.yaml

```diff
@@ -0,0 +1,68 @@
+name: "CI"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master 
+  create:
+    tags:
+      - v*
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  cancel:
+    name: "Cancel"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Get workflow id"
+        uses: styfle/cancel-workflow-action@0.6.0
+        with:
+          access_token: ${{ github.token }}
+
+  build:
+    name: "Build"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+    steps:
+
```

Comment:
I see that "checkout" step is no longer here -> it proved redundant?

Reply:
Nope, I just made a mess lol (fixed in the latest commit).

Good catch!